src/devices/RotaryEncoder/QuadratureRotaryEncoder.cs

```diff
@@ -35,12 +35,20 @@ public class QuadratureRotaryEncoder
         public float Rotations { get => (float)PulseCount / PulsesPerRotation; }
 
         /// <summary>The DebounceMilliseconds property represents the minimum amount of delay allowed between falling edges of the A (clk) pin.</summary>
-        public uint DebounceMilliseconds { get; set; } = 5;
+        public TimeSpan DebounceMilliseconds
```

Comment:
DebounceTime since unit is part of the type now (make sure to update the doc to mention recommended unit)

Reply:
fixing